### PR TITLE
Grammar Fix Challenge Create

### PR DESCRIPTION
**Pull request by trexdinosaur**
Very minor grammar changes.

### CODE REVIEW

File: frontend/src/views/web/challenge-create.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <div class="row">
         <div class="col s12 m9 offset-m3">
             <i class="fa fa-info-circle"></i>
-            To know how to create a challenge using zip configuration, please see our documentation
+            To learn how to create a challenge using zip file configuration, please see our documentation
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge-create.html]: @trexdinosaur I think `know` makes better sense here. Please revert this change.

**Comment by trexdinosaur** [comment]
Updated
